packages/forms/frontend/sirius-components-forms/src/groups/Group.tsx

```diff
@@ -69,17 +69,6 @@ const useGroupStyles = makeStyles<Theme, GroupStyleProps>((theme) => ({
     paddingTop: '1px',
     paddingBottom: '0px',
   },
-  buttonIcon: {
-    marginRight: theme.spacing(1),
```

Comment:
We're losing the `marginRight: theme.spacing(1)`. Is that correct?

Reply:
I added the possibility to custom icon style.
Done